boss-ws/src/test/resources/boss-config.yml

```diff
@@ -11,19 +11,22 @@ database:
   password: ${dataSource.password}
   url: ${dataSource.url}
   validationQuery: ${dataSource.validationQuery}
-cloudStore:
-  username: ignoredInUnitTesting
-  password: ignoredInUnitTesting
-  bucket: bossbucket
-  endpoint: http://localhost:8180/fcs
-  type: FCS
-localStore:
-  username: ecsUserName
-  password: ecsPassword
-  bucket: bossbucket
-  endpoint: https://ecsserver.broadinstitute.org
-  pathStyleAccess: true
-  type: S3
+objectStores:
```

Comment:
We really only need one object store for unit testing -- we could call it "mockStore".  The unit tests will need to be changed to use this.